pages/Documents/MessagingChannels/PushNotificationService/Configuration/push-proxy.md

```diff
@@ -30,17 +32,24 @@ In order to enable the push proxy feature, you will require the following data a
 
 ### Optional configurations
 
-* **Authorization**: HTTP Basic Authentication (BA) header.
+#### Basic Auth
+
+* HTTP Basic Authentication (BA) header.
 
     * Username. _Example_:  Alibaba.
 
     * Password. _Example_: opensesame.
 
+
+#### Mutual TLS ([TLS Authentication](../tls-authentication.md))
```

Comment:
Does this link work? Consider linking to the “permalink” (it seems to be `(push-notification-service-tls-authentication.html)`).